nedlibreader/time_test.go

```diff
@@ -0,0 +1,78 @@
+package nedlibreader
+
+import (
+	"testing"
+	"time"
+)
+
+type testTime struct {
+	timeAsString string
+	time         time.Time
+	shouldFail   bool
+}
+
+type testTable struct {
+	expectedFormat string
+	tests          []testTime
+}
+
+func TestParseTime(t *testing.T) {
+	var testTable = []testTable{
```

Comment:
The table should be a global variable such that it can be reused by other tests.

Reply:
What other tests? This code can be moved when such tests appear.